### PR TITLE
Use react router layouts

### PR DESCRIPTION
**Pull request by mzhubail**

### CODE REVIEW

File: packages/frontend/src/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  return (
+    <main className="bg-grey-1 min-h-screen">
+      <Nav />
+      <div className={containerClasses}>
+        <Outlet />
+      </div>
+    </main>
+  );
+};
```
Comment (mzhubail) [inline packages/frontend/src/Layout.tsx:21]: Here, `<Outlet />` will be replaced by whatever inside the page.  If you go to writing task 1 page, the content of the page will be placed in place of `<Outlet />`, and same goes for other pages.

This is very helpful, as now, not every page has to include `Nav` and include padding to the screen.

File: packages/frontend/src/main.tsx
Comment (mzhubail) [inline packages/frontend/src/main.tsx:1]: Note that for some pages I didn't use the layout.

@DanaK270 the nav bar conflict with the nav included in your page.

**Comment by ThatIsHero** [comment]
Has been reviewed and everything works will

can't marge it as I considered last pusher (Solving conflict)

Ready to merge !

@Ma-hawaj

**Review by Ma-hawaj — APPROVED** [review]
Approved on behalf of @ThatIsHero

**Pull request merged**